.all-contributorsrc

```diff
@@ -247,6 +247,13 @@
       "contributions": [
         "bug"
       ]
+    },
+    {
+      "login": "Garbee",
+      "name": "Jonathan Garbee",
+      "avatar_url": "https://avatars.githubusercontent.com/u/868301?v=3",
+      "profile": "http://jonathan.garbee.me",
+      "contributions": []
```

Comment:
Please try again and if for some reason it doesn't work, do let us know. Maybe there is another issue? Thanks!